src/Task/PhpCsFixerV2.php

```diff
@@ -72,7 +73,13 @@ public function run(ContextInterface $context)
         $arguments->addOptionalArgument('--allow-risky=%s', $config['allow_risky'] ? 'yes' : 'no');
         $arguments->addOptionalArgument('--cache-file=%s', $config['cache_file']);
         $arguments->addOptionalArgument('--config=%s', $config['config']);
-        $arguments->addOptionalCommaSeparatedArgument('--rules=%s', $config['rules']);
+
+        if (is_array($rules = $config['rules']) && ArrayType::isMap($rules)) {
```

Comment:
`rules` is always an array as enforced by the options resolver.
Can you show me some example on how to configure the rules parameter?